btrack/io/_localization.py

```diff
@@ -120,13 +123,13 @@ def _concat_centroids(centroids, new_centroids):
     return centroids
 
 
-def segmentation_to_objects(
+def segmentation_to_objects(  # noqa: PLR0913
     segmentation: Union[np.ndarray, Generator],
     intensity_image: Optional[Union[np.ndarray, Generator]] = None,
     properties: Optional[Tuple[str]] = (),
     scale: Optional[Tuple[float]] = None,
-    use_weighted_centroid: bool = True,
-    assign_class_ID: bool = False,
+    use_weighted_centroid: bool = True,  # noqa: FBT001 FBT002
+    assign_class_ID: bool = False,  # noqa: FBT001 FBT002
```

Comment:
I think the usual syntax is this
```suggestion
    use_weighted_centroid: bool = True,  # noqa: FBT001,FBT002
    assign_class_ID: bool = False,  # noqa: FBT001,FBT002
```